### DISCUSSION TITLE
Missing functionality of the mobile app

### DISCUSSION THREAD

**Discussion by mdziczkowski** [community/community#84147] [Mobile]
### Select Topic Area

Product Feedback

### Body

**OS**: Android 13

In the mobile app are missing many features avaliable thru website and pc, like for example "fork", "download", etc.

**Comment by imanmahjoubi** [comment]
@mdziczkowski Thank you for sharing your feedback with us. We understand your concern about missing features such as "fork" and "download" that are available on the website and Desktop. We're actively working on improving the mobile app to bring a more comprehensive experience closer to the functionality offered on other platforms.

**Reply by mdziczkowski** [reply]
Hello. I was suprised that such functionality wasn't already build-in to provide better comtability  with the website version.

Such functionality as on GitHub website beeing already build-in could realy make the work with code much easier 😉

**Reply by Gnezdilov66** [reply]
[]() the best

**Reply by akemmanuel** [reply]
Thank you! It is the same in the IOS app...

**Comment by jsoref** [comment]
https://github.com/orgs/community/discussions/3333 asked for fork. https://twitter.com/github/status/1866883185742622746 claims the feature exists.

Not sure about Download, but amorphous tickets are really hard to mark as "Done".
